tests/PackageSecurityExtensionTest.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+class PackageSecurityExtensionTest extends SapphireTest
+{
+    protected static $fixture_file = 'PackagesWithAlerts.yml';
+
+    public function setUp()
+    {
+        parent::setUp();
+        $manifest = new SS_ConfigManifest(BASE_PATH, true);
+        $moduleDoesNotExist = $manifest->moduleExists('bringyourownideas/silverstripe-maintenance');
```

Comment:
This seems like excessive boilerplate - the standard is to use `class_exists`. You will also need to run these checks before `parent::setUp()` otherwise you'll get exceptions from the fixture factory when trying to load non-existent fixture classes

Reply:
Neat, ta.